lib/ip.js

```diff
@@ -376,7 +376,22 @@ ip.address = function(name, family) {
     return res[0].address;
   }
 
-  var all = Object.keys(interfaces).map(function (nic) {
+  function priority(name) {
+    if (name.substring(0, 2) === 'en' || name.substring(0, 3) === 'eth') {
```

Comment:
Sorry for delay. I've a suggestion, what do you think about either using `slice` here or RegExp?

Reply:
@indutny `slice` works for me!